apps/wordlebyt/wordlebyt.go

```diff
@@ -0,0 +1,25 @@
+// Package wordlebyt provides details for the Wordlebyt applet.
+package wordlebyt
+
+import (
+	_ "embed"
+
+	"tidbyt.dev/community/apps/manifest"
+)
+
+//go:embed wordlebyt.star
+var source []byte
+
+// New creates a new instance of the Wordlebyt applet.
+func New() manifest.Manifest {
+	return manifest.Manifest{
+		ID:          "wordlebyt",
+		Name:        "Wordlebyt",
+		Author:      "skola28",
+		Summary:     "Display daily Wordle score",
+		Desc:        "Show off your daily Wordle Score.",
```

Comment:
The schema field description is currently not visible in the iOS/Android app. Would be great if the app description could explain how to copy/paste the score.